src/lib/layouts/DocsTutorial.svelte

```diff
@@ -3,77 +3,103 @@
     import { Feedback } from '$lib/components';
     import type { Tutorial } from '$markdoc/layouts/Tutorial.svelte';
     import type { TocItem } from './DocsArticle.svelte';
+    import Heading from '$markdoc/nodes/Heading.svelte';
+    import { onMount } from 'svelte';
 
-    export let title: string;
     export let toc: Array<TocItem>;
     export let currentStep: number;
-    export let back: string;
     export let date: string;
 
     export let tutorials: Array<Tutorial>;
 
+    const firstStepItem = tutorials[0];
```

Comment:
If tutorials is an empty array, this would yield a nullpointer right?

Reply:
Possibly yes, although I doubt it'll be empty as there's each for a framework, split into different steps. Wdyt?

Reply:
I would prefer to check it, just so we're sure that if something goes wrong upstream, it doesn't crash the site.